therapy/etl/base.py

```diff
@@ -24,7 +24,7 @@ def __init__(self, database, data_path=PROJECT_ROOT / 'data'):
         name = self.__class__.__name__.lower()
         self.database = database
         self._src_data_dir = data_path / name
-        self.in_normalize = name in ACCEPTED_SOURCES
+        self.in_normalize = name
```

Comment:
sorry to make you continue to unwind this, but this also can be removed (there is a line somewhere in therapy.cli that uses it -- we can pull that check as well)

`name in ACCEPTED SOURCES` is a boolean value that is used as a class variable to indicate that this class should be included *in normalize*

`name` is just the name (without the `in` operator it doesn't get cast to a bool)